### PR TITLE
Saving request remote_ip on TicketGrantingTicket user_ip

### PR DESCRIPTION
**Pull request by alepaez**

### CODE REVIEW

File: app/processors/casino/ticket_granting_ticket_processor.rb [hunk 1]
```diff
@@ -25,13 +25,14 @@ def find_valid_ticket_granting_ticket(ticket, user_agent, options = {})
...
 
-  def acquire_ticket_granting_ticket(authentication_result, user_agent, options = {})
+  def acquire_ticket_granting_ticket(authentication_result, user_agent, user_ip, options = {})
     user_data = authentication_result[:user_data]
     user = load_or_initialize_user(authentication_result[:authenticator], user_data[:username], user_data[:extra_attributes])
     cleanup_expired_ticket_granting_tickets(user)
     user.ticket_granting_tickets.create!({
       awaiting_two_factor_authentication: !user.active_two_factor_authenticator.nil?,
       user_agent: user_agent,
+      user_ip: user_ip,
```
Comment (luxflux) [inline app/processors/casino/ticket_granting_ticket_processor.rb]: Could you add a test to cover the changes in this file?

**Comment by luxflux** [comment]
Thank you for your work! It looks quite good. Could you check my comment? Afterwards we are ready to merge, I think.

**Comment by alepaez** [comment]
Done!

**Comment by pencil** [comment]
LGTM

**Pull request merged**